lib/archethic_web/views/explorer_view.ex

```diff
@@ -199,10 +199,10 @@ defmodule ArchethicWeb.ExplorerView do
     """
   end
 
-  def format_transaction_content(:origin_shared_secrets, content) do
+  def format_transaction_content(:origin, content) do
     get_origin_public_keys(content, %{software: [], hardware: []})
```

Comment:
I think you have to change this function as it's including the certificate as well.
Also this function have only one key now